packages/2d/src/curves/CurvePoint.ts

```diff
@@ -2,5 +2,13 @@ import type {Vector2} from '@motion-canvas/core/lib/types';
 
 export interface CurvePoint {
   position: Vector2;
+  /**
+   * @deprecated The tangent is currently inconsistent for different types of
```

Comment:
When used as a block tag, `@deprecated` should be formatted with a new line:
```suggestion
   * @deprecated 
   * The tangent is currently inconsistent for different types of
```